### DISCUSSION TITLE
Welcome to Grav Discussions!

### DISCUSSION THREAD

**Discussion by w00fz** [getgrav/grav#3096] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PeopleInside** [comment]
Hi, my name is Marco and I'm discovering Grav.
The community will be useful for question about issues and doubt.

I'm a web, security and open source passionate.

**Reply by w00fz** [reply]
Welcome Marco!
